### PR TITLE
Contact Mailing Address Email Template Issue Fixed

### PR DESCRIPTION
**Pull request by asadmanzoor93**
**Description:** This PR updates `CONTACT_MAILING_ADDRESS` field in site configuration we override it in email templates (password reset, enrollment mails from instructor dashboard)

**Merge checklist:**

- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are (reasonably) squashed
- [ ] Translations are updated
- [ ] PR author is listed in AUTHORS

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

**Note:** Please add screenshots for any viewable change.

### CODE REVIEW

File: openedx/core/djangoapps/ace_common/template_context.py [hunk 1]
```diff
@@ -50,8 +59,7 @@ def get_base_template_context(site):
         ),
         'contact_email': get_config_value_from_site_or_settings(
             'CONTACT_EMAIL', site=site, site_config_name='contact_email'),
-        'contact_mailing_address': get_config_value_from_site_or_settings(
```
Comment (iam-mhaseeb) [inline openedx/core/djangoapps/ace_common/template_context.py:53]: `get_config_value_from_site_or_settings` is not picking the right website which is weird but for now, we can live with it.

File: openedx/core/djangoapps/ace_common/template_context.py [hunk 2]
```diff
@@ -38,6 +38,15 @@ def get_base_template_context(site):
     except NoReverseMatch:
         dashboard_url = reverse('home')
 
+    contact_mailing_address = configuration_helpers.get_value('CONTACT_MAILING_ADDRESS1')
```
Comment (burhandodhy) [inline openedx/core/djangoapps/ace_common/template_context.py]: Is it typo `CONTACT_MAILING_ADDRESS1` or there is a specific reason to add the 1 ?

Comment (asadmanzoor93) [reply openedx/core/djangoapps/ace_common/template_context.py]: typo.. already removed.

Comment (iam-mhaseeb) [reply openedx/core/djangoapps/ace_common/template_context.py]: It's fixed in latest commit.

**Pull request merged**